python/app/model_entity_listing.py

```diff
@@ -190,3 +196,50 @@ def _populate_thumbnail_image(self, item, field, image, path):
         sg_data = item.get_sg_data()
         icon = self._sg_formatter.create_thumbnail(image, sg_data)
         item.setIcon(QtGui.QIcon(icon))
+
+    def _before_data_processing(self, data):
```

Comment:
I just found out about the `return_paging_info_without_counts` argument available in the `apiv3/json`.
Unfortunately, this argument is hardcoded to False in the _python-api_.
I don't know yet if it is possible to set it for this request.